authentication/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+module life.qbic.authentication {
+  requires org.slf4j;
+  requires java.persistence;
+  requires life.qbic.broadcasting;
+
+  exports life.qbic.authentication.domain.user.concept;
+  exports life.qbic.authentication.application;
+  exports life.qbic.authentication.domain.user.repository;
+  exports life.qbic.authentication.domain.user.event;
+  exports life.qbic.authentication.domain.user.policy;
+  exports life.qbic.authentication.application.user.registration;
+  exports life.qbic.authentication.application.user.password;
+  exports life.qbic.authentication.application.notification;
```

Comment:
Do you need to specify those if you already export the parent package of all of those?

Reply:
see line 7

Reply:
Sub-packages are not included in the export - All exports are specific to the package level that is defined.